packages/core/database/encryption/README.md

```diff
@@ -146,11 +154,129 @@ const ENCRYPTION_SCHEMA = {
 };
 ```
 
+**Note**: The core schema now includes common authentication fields for OAuth, API key, and basic authentication. API modules should use `api_key` (snake_case) in their `apiPropertiesToPersist.credential` arrays for consistency with OAuth2Requester and BasicAuthRequester conventions.
+
+### API Module Credential Naming Conventions
+
+When creating API module definitions, use **snake_case** for credential property names to ensure automatic encryption:
+
+**✅ Recommended (automatically encrypted):**
+```javascript
+// API Module Definition
+const Definition = {
+    requiredAuthMethods: {
+        apiPropertiesToPersist: {
+            credential: ['api_key'],           // ✅ Automatically encrypted
+            credential: ['access_token', 'refresh_token'],  // ✅ OAuth - encrypted
+            credential: ['username', 'password'],           // ✅ Basic auth - encrypted
```

Comment:
Please add a `//or` in between the "credential" prop, it was confusing to me at first sight having so many credentials props in  `apiPropertiesToPersist`